sdk/ai/ai-voicelive/simple-dev.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
```

Comment:
In general there's a lot of scripts to maintain here with maybe some opportunities to centralize / dedupe